templates/vue/src/utils/dragSelectModular.js

```diff
@@ -51,4 +51,13 @@ export default class DragSelectModular {
   static pauseDragSelect() {
     DragSelectModular.dragSelect.break()
   }
+
+  static initializeDragSelectZone(mainElementId) {
+    let allowedArea = document.getElementById(mainElementId)
+    document.body.addEventListener("mousedown", function(event) {
+      if (!allowedArea.contains(event.target)) {
+        DragSelectModular.pauseDragSelect()
+      }
+    })
+  }
```

Comment:
Can't you just use the `area` parameter to set the drag area? https://thibaultjanbeyer.github.io/DragSelect/DragSelect.html

Reply:
Alternatively, I think there's also a `vue-drag-slider` package that we could use instead that automatically has a wrapper for the area.

Reply:
I decided to use your first suggestion!